ui/client/containers/Archive.tsx

```diff
@@ -18,7 +18,8 @@ const ElementsRenderer: RowsRenderer = ({processes}) => processes.map(process =>
     <Td column="category">{process.processCategory}</Td>
     <Td column="createdBy" className="centered-column" value={process.createdBy}>{process.createdBy}</Td>
     <Td column="createdAt" className="centered-column" value={process.createdAt}><Date date={process.createdAt}/></Td>
-    <Td column="modifyDate" className="centered-column" value={process.modificationDate}><Date date={process.modificationDate}/></Td>
+    <Td column="actionDate" className="centered-column" value={process?.lastAction?.performedAt}><Date date={process?.lastAction?.performedAt}/></Td>
```

Comment:
Is it obvious that for archived processes last action is Archive? Now it is so, but if we add other actions??

Reply:
Right now we assume that after archive we can do only unarchvie :) If want change this behavior in future, then we should return this information in another fields.  Right now we can leave it.